projects/editions/pkg/install-dfm/Makefile

```diff
@@ -0,0 +1,33 @@
+default=tag
+ORG?=edtestorg
+IMAGE=install-dfm
+DEPS=Dockerfile build/dfm/port build/dfm/docker
+
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+
+# Create empty dir for /port mount
+build/dfm/port: Makefile
+	mkdir -p build/dfm/port
```

Comment:
don't create host directories, ownership will be all wrong, create them in the Dockerfile